ERROR 2022-07-15 18:04:44,230 - Error Message from get_list_item
Traceback (most recent call last):
  File "d:\Tinghao.Chen\Desktop\oneTouch_tool\test_plan_ui.py", line 407, in get_list_item
    self.renew_ini()
AttributeError: 'getSHA' object has no attribute 'renew_ini'
